src/main/kotlin/ai/digital/integration/server/deploy/internals/cluster/DeployDockerClusterHelper.kt

```diff
@@ -265,16 +279,24 @@ open class DeployDockerClusterHelper(val project: Project) : DockerClusterHelper
     }
 
     private fun getMasterIp(order: Int): String {
-        try {
-            return DockerUtil.inspect(project,
-                "{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}",
-                "cluster_xl-deploy-master_${order}")
-        } catch (e: ExecException) {
-            // fallback in naming
-            return DockerUtil.inspect(project,
-                "{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}",
-                "cluster-xl-deploy-master-${order}")
+        val instanceIds = listOf(
+            "cluster-xl-deploy-master-${order}",
+            "cluster_xl-deploy-master_${order}"
+        )
```

Comment:
Why do our instanceIDs have two different masters?

Reply:
sometimes it is creating with _ and some times with - , to cover both cases had to add this, before there used to be a try catch for both scenarios

1) cluster-xl-deploy-master-
2)cluster_xl-deploy-master_